src/main/java/dk/kb/image/kaltura/DsKalturaClient.java

```diff
@@ -0,0 +1,89 @@
+package dk.kb.image.kaltura;
+
+import java.util.ArrayList;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.kaltura.client.KalturaApiException;
+import com.kaltura.client.KalturaClient;
+import com.kaltura.client.KalturaConfiguration;
+import com.kaltura.client.enums.KalturaSessionType;
+import com.kaltura.client.types.KalturaFilterPager;
+import com.kaltura.client.types.KalturaMediaEntry;
+import com.kaltura.client.types.KalturaMediaEntryFilter;
+import com.kaltura.client.types.KalturaMediaListResponse;
+
+/**
+ * Kaltura client that can: 
+ * 1) API lookup and map external ID to internal ID 
+ */
+public class DsKalturaClient {
+
+    private KalturaClient client = null;
+    private static final Logger log = LoggerFactory.getLogger(DsKalturaClient.class);
+
+    /**
+     * Instantiate a session to Kaltura that can be used to upload files. The session can be be reused for multiple uploads etc.
+     * 
+     * @param kalturaUrl The Kaltura API url. Using the baseUrl will automatic append the API service part to the URL. 
+     * @param userId The userId that must be defined in the kaltura, userId is email xxx@kb.dk in our kaltura
+     * @param partnerId The partner id for kaltura. Kind of a collectionId. 
+     * @param adminSecret The admin secret that must not be shared that gives access to API
+     */
+    public DsKalturaClient(String kalturaUrl, String userId, int partnerId, String adminSecret) throws Exception {
+        try {
+            KalturaConfiguration config = new KalturaConfiguration();
+            config.setEndpoint(kalturaUrl);
+            KalturaClient client = new KalturaClient(config);
+            String ks = client.generateSession(adminSecret, userId, KalturaSessionType.ADMIN, partnerId);
+            client.setKs(ks);
+            this.client=client;
+        }
+        catch (Exception e) {
+            log.error("Connecting to Kaltura failed for kaltura url:"+kalturaUrl,e.getMessage());
```

Comment:
Same comment regarding the log level as for the `log.error` in `AccessApiServiceImpl`.

Reply:
downgraded to warn as well.